libs/ui/src/precinct_scanner_tally_report.tsx

```diff
@@ -21,68 +21,67 @@ import {
   TallyReportColumns,
 } from './tally_report';
 import { TallyReportSummary } from './tally_report_summary';
-import { Text } from './text';
 
 const Header = styled.div`
-  margin-bottom: 0.75em;
-
   & h1 {
     margin-top: 0;
     margin-bottom: 0.5em;
     font-size: 1.5em;
   }
 
-  & h2 {
-    margin-top: 0;
-    margin-bottom: 0.15em;
-    font-size: 1.25em;
-  }
-
   & p {
     margin-top: 0;
     margin-bottom: 0.25em;
   }
 `;
 
-const SignatureContainer = styled.div`
-  margin: 1em 0;
-  & p {
-    margin-bottom: 1.5em;
+const HeaderData = styled.span`
+  margin-right: 1em;
+  white-space: nowrap;
+  &:last-child {
+    margin-right: 0;
   }
 `;
 
-const SignatureLine = styled.div`
-  display: flex;
-  justify-content: space-around;
+const ReportCertificationSignaturesContainer = styled.div`
+  margin-top: 1em;
 `;
 
-const SignatureSpace = styled.span`
-  border-bottom: 1px solid #000000;
-  width: 30%;
-  padding-bottom: 1px;
-  &::before {
-    font-family: 'Noto Emoji', sans-serif;
-    font-size: 1.5em;
-    content: '⨉';
+const Signatures = styled.div`
+  display: flex;
+  & > div {
+    flex: 1;
```

Comment:
This is neat, I didn't know you could do this.

Reply:
Cool. Glad that I could share something new. 

Note: One benefit of this implementation is that we can easily change the number of signature lines by simply adding/removing child div elements. We could have a election config value for `pollsReportCertificationSignatures` where `0` disables the feature, and a value N adds N signature lines.